internal/controllers/aggregation/status.go

```diff
@@ -57,29 +59,52 @@ func (s *statusController) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		}
 
 		for _, state := range slice.Status.Resources {
+			state := state
```

Comment:
We can thankfully say goodbye to this idiom in 1.22!

Reply:
YESSSS